File: backend/src/main/java/com/entvy/taste/entity/TastePlace.java
package com.entvy.taste.entity;

import jakarta.persistence.*;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import java.time.LocalDateTime;

@Entity
@Table(name = "taste_place")
@Getter
@Setter
@NoArgsConstructor
public class TastePlace {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    private String sigunNm;
    private String sigunCd;
    private String restrtNm;
    private String reprsntFoodNm;
    private String tastfdplcTelno;
    private String refineLotnoAddr;
    private String refineRoadnmAddr;
    private String refineZipCd;
    private Double refineWgs84Logt;
    private Double refineWgs84Lat;

    @Column(updatable = false)
    private LocalDateTime createdAt = LocalDateTime.now();
}

File: frontend/src/types/tastePlace.ts
export interface TastePlace {
  restrtNm: string; // 음식점명
  refineRoadnmAddr: string; // 도로명주소
  tastfdplcTelno: string; // 전화번호
  refineLotnoAddr?: string; // 구주소
  refineWgs84Lat?: number; // 위도
  refineWgs84Logt?: number; // 경도
  sigunNm?: string; // 시군명
  sigunCd?: string; // 시군코드
  reprsntFoodNm?: string; // 대표음식명
  refineZipCd?: string; // 소재지우편번호
}